FEATURES.md

```diff
@@ -46,4 +46,12 @@ Namespaces are a way to organize clusters into virtual sub-clusters — they can
 ![Releases4](images/screenshot_release4.png)
 
 Updated" refers to the amount of time that has passed since the last revision of the release. Whenever you install or upgrade the release, a new revision is created. You can think of it as the "age" of the latest revision.
-![Releases5](images/screenshot_release5.png)
\ No newline at end of file
+![Releases5](images/screenshot_release5.png)
+
+# Release details
```

Comment:
It's good for the start. 

I would mention the block on the left is called "Revision History". It displays the list of past release revisions, with the current revision on top. And each item in it shows status, age and version upgrade/downgrade.

Reply:
Also, the screenshot of screenshot_release_detail1.png has much better revision history illustration.